pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -573,6 +584,7 @@ func (p *PriorityQueue) Update(oldPod, newPod *v1.Pod) error {
 		return err
 	}
 	p.PodNominator.AddNominatedPod(pInfo.PodInfo, nil)
+	klog.V(5).InfoS("Pod moved", "pod", klog.KObj(pInfo.Pod), "event", "PodUpdated", "source", unschedulablePods, "to", activeQName)
```

Comment:
the event value should be a constant, like the others.

Why does it say from unschedulable, if the pod was not there (line 581).

Reply:
I think the "source" info can be laggy (in AddUnschedulabelIfNotPresent)  or inaccurate (this case). In one offline discussion with Yuan, I was sort of keen on logging the "destination" queue name only since you can infer the "source" from the previous log. WDYT?

Reply:
> the event value should be a constant, like the others.
Will make the change.

Reply:
+1 on only the destination

Reply:
> I think the "source" info can be laggy (in AddUnschedulabelIfNotPresent) or inaccurate (this case). In one offline discussion with Yuan, I was sort of keen on logging the "destination" queue name only since you can infer the "source" from the previous log. WDYT?

I think @alculquicondor's comment was the pod was not in any queue in this event. I changed `source` to `notInQueue`. Also, I'd suggest we keep `source` for readability.

Reply:
> +1 on only the destination

OK, removed `from` and `source` from the events.